client/app/components/dashboards/widget.html

```diff
@@ -29,6 +29,12 @@
             <li ng-if="$ctrl.dashboard.canEdit()"><a ng-click="$ctrl.deleteWidget()">Remove from Dashboard</a></li>
           </ul>
         </div>
+        <div id="topRefreshIndicator" ng-if="$ctrl.widget.loading">
```

Comment:
Isn't `id` supposed to be singular on a page?

Reply:
Only when retrieving elements by id but in terms of css hooks, it has no significance.

I'll change it though 👍🏻